mobile_verifier/src/reward_shares.rs

```diff
@@ -134,6 +133,66 @@ impl TransferRewards {
     }
 }
 
+#[derive(Default)]
+pub struct MapperShares {
+    pub discovery_mapping_shares: Vec<Vec<u8>>,
```

Comment:
```suggestion
    pub discovery_mapping_shares: SubscriberLocationMap,
```
reuse the same "type" everywhere, even if it's just an alias. also we should think about renaming this type/alias name since it's no longer a map. `SubscriberValidatedLocations`?